core/src/main/res/values/strings.xml

```diff
@@ -516,7 +516,7 @@
     <string name="stop_playback">Stop playback</string>
     <string name="continue_playback">Continue audio playback</string>
     <string name="behavior">Behavior</string>
-    <string name="pref_back_button_behavior_title">Back Button Behavior</string>
+    <string name="pref_back_button_behavior_title">\"Back\" Button Behavior</string>
```

Comment:
Why should this be in quotes? Is it not _really_ the back button?

Reply:
It cites an actual button.